app/wallet.py

```diff
@@ -144,7 +138,7 @@ def sendPayment(self, address, value, assets):
   pk = tree[6:72]
   fromPk = Wallet.fromPk(pk, network).publicKey()
   isValid = Wallet.isValid()
-  logging.info(f"""Validation:
+  print(f"""Validation:
```

Comment:
should this not be logger.info?

Reply:
- we really shouldn't call __main__ at all, but I left it there as an example.  
- also, this isn't a route call, so it doesn't really address anything in the API